infra/dss-events-scribe/main.tf

```diff
@@ -0,0 +1,76 @@
+data aws_caller_identity current {}
+data aws_region current {}
+
+locals {
+  region = "${data.aws_region.current.name}"
+  account_id = "${data.aws_caller_identity.current.account_id}"
+  common_tags = "${map(
+    "managedBy" , "terraform",
+    "Name"      , "${var.DSS_INFRA_TAG_SERVICE}-subscriptionsdynamodb",
+    "project"   , "${var.DSS_INFRA_TAG_PROJECT}",
+    "env"       , "${var.DSS_DEPLOYMENT_STAGE}",
+    "service"   , "${var.DSS_INFRA_TAG_SERVICE}",
+    "owner"     , "${var.DSS_INFRA_TAG_OWNER}"
+  )}"
+}
+
+locals {
+  replicas = ["aws", "gcp"]
+}
+
+data "aws_iam_policy_document" "sqs" {
+  statement {
+    principals {
+      type = "AWS"
+      identifiers = ["*"]
+    }
+    actions = ["sqs:SendMessage"]
+    resources = ["arn:aws:sqs:${local.region}:${local.account_id}:dss-events-scribe-${var.DSS_DEPLOYMENT_STAGE}"]
+    condition {
+      test     = "StringEquals"
+      variable = "aws:SourceArn"
+      values = ["${aws_cloudwatch_event_rule.events-scribe.arn}"]
```

Comment:
Just to make sure I understand correctly - this policy gives any cloudwatch event rule-based principal permission to send a message to the SQS queue used for event journaling?

Reply:
yep